#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Localedata update.
# DP: Related bugs: 
# DP: Dpatch author: GOTO Masanori <gotom@debian.org>
# DP: Patch author: Various
# DP: Upstream status: In CVS
# DP: Status Details: 
# DP: Date: 2005-07-17

PATCHLEVEL=0

if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p$PATCHLEVEL < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p$PATCHLEVEL < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

# append the patch here and adjust the -p? flag in the patch calls.
--- localedata/ChangeLog.orig	2005-07-17 22:52:16.172301704 +0900
+++ localedata/ChangeLog	2005-07-17 22:54:26.456842444 +0900
@@ -1,8 +1,52 @@
+2005-07-08  Ulrich Drepper  <drepper@redhat.com>
+
+	[BZ #408]
+	* locales/vi_VN: Update locale.
+
+2005-07-05  Ulrich Drepper  <drepper@redhat.com>
+
+	* locales/en_US: Remove duplicate am_pn entry.
+
+2005-05-28  Jakub Jelinek  <jakub@redhat.com>
+
+	[BZ #622]
+	* locales/pa_IN (am_pm): Fix typos.
+	Patch by Denis Barbier <barbier@linuxfr.org>.
+
+2005-05-22  Ulrich Drepper  <drepper@redhat.com>
+
+	* locales/mn_MN: Update.  Patch by Sanlig Badral <s_badral@yahoo.com>.
+
+2005-04-28  Jakub Jelinek  <jakub@redhat.com>
+
+	* SUPPORTED: Add UTF-8 locales where UTF-8 was not already supported.
+
+2005-04-26  Ulrich Drepper  <drepper@redhat.com>
+
+	* locales/fa_IR: Add alt_digits, change date and time
+	representation, and various cleanups.
+	Patch by Hamed Malek <hamed@bamdad.org>.
+
 2005-03-21  Jakub Jelinek  <jakub@redhat.com>
 
 	[BZ #823]
 	* charmaps/WINDOWS-31J: Add % before alias keyword.
 
+2005-02-27  Denis Barbier  <barbier@debian.org>
+
+	[BZ #38]
+	* locales/sr_CS: New file.
+	Contributed by Danilo Segan <dsegan@gmx.net>
+	* SUPPORTED: Add sr_CS/ISO-8859-5 and sr_CS.UTF-8/UTF-8.
+
+2005-03-18  Andreas Schwab  <schwab@suse.de>
+
+	* locales/pa_IN (am_pm): Fix character names.
+
+2005-03-18  Ulrich Drepper  <drepper@redhat.com>
+
+	* SUPPORTED (SUPPORTED-LOCALES): Add af_ZA.UTF-8.
+
 2004-12-19  Roland McGrath  <roland@frob.com>
 
 	* gen-unicode-ctype.c (output_tables): Fix email address in output.
Index: localedata/SUPPORTED
===================================================================
RCS file: /cvs/glibc/libc/localedata/SUPPORTED,v
retrieving revision 1.72
retrieving revision 1.75
diff -u -p -r1.72 -r1.75
--- localedata/SUPPORTED	31 Oct 2004 20:49:54 -0000	1.72
+++ localedata/SUPPORTED	28 Apr 2005 14:21:30 -0000	1.75
@@ -1,12 +1,15 @@
 # This file names the currently supported and somewhat tested locales.
 # If you have any additions please file a glibc bug report.
 SUPPORTED-LOCALES=\
+aa_DJ.UTF-8/UTF-8 \
 aa_DJ/ISO-8859-1 \
 aa_ER/UTF-8 \
 aa_ER@saaho/UTF-8 \
 aa_ET/UTF-8 \
+af_ZA.UTF-8/UTF-8 \
 af_ZA/ISO-8859-1 \
 am_ET/UTF-8 \
+an_ES.UTF-8/UTF-8 \
 an_ES/ISO-8859-15 \
 ar_AE.UTF-8/UTF-8 \
 ar_AE/ISO-8859-6 \
@@ -50,8 +53,10 @@ bg_BG.UTF-8/UTF-8 \
 bg_BG/CP1251 \
 bn_BD/UTF-8 \
 bn_IN/UTF-8 \
+br_FR.UTF-8/UTF-8 \
 br_FR/ISO-8859-1 \
 br_FR@euro/ISO-8859-15 \
+bs_BA.UTF-8/UTF-8 \
 bs_BA/ISO-8859-2 \
 byn_ER/UTF-8 \
 ca_ES.UTF-8/UTF-8 \
@@ -176,6 +181,7 @@ fr_LU@euro/ISO-8859-15 \
 ga_IE.UTF-8/UTF-8 \
 ga_IE/ISO-8859-1 \
 ga_IE@euro/ISO-8859-15 \
+gd_GB.UTF-8/UTF-8 \
 gd_GB/ISO-8859-15 \
 gez_ER/UTF-8 \
 gez_ER@abegede/UTF-8 \
@@ -207,7 +213,9 @@ iw_IL.UTF-8/UTF-8 \
 iw_IL/ISO-8859-8 \
 ja_JP.EUC-JP/EUC-JP \
 ja_JP.UTF-8/UTF-8 \
+ka_GE.UTF-8/UTF-8 \
 ka_GE/GEORGIAN-PS \
+kk_KZ.UTF-8/UTF-8 \
 kk_KZ/PT154 \
 kl_GL.UTF-8/UTF-8 \
 kl_GL/ISO-8859-1 \
@@ -216,12 +224,14 @@ ko_KR.EUC-KR/EUC-KR \
 ko_KR.UTF-8/UTF-8 \
 kw_GB.UTF-8/UTF-8 \
 kw_GB/ISO-8859-1 \
+lg_UG.UTF-8/UTF-8 \
 lg_UG/ISO-8859-10 \
 lo_LA/UTF-8 \
 lt_LT.UTF-8/UTF-8 \
 lt_LT/ISO-8859-13 \
 lv_LV.UTF-8/UTF-8 \
 lv_LV/ISO-8859-13 \
+mi_NZ.UTF-8/UTF-8 \
 mi_NZ/ISO-8859-13 \
 mk_MK.UTF-8/UTF-8 \
 mk_MK/ISO-8859-5 \
@@ -243,8 +253,10 @@ nl_NL/ISO-8859-1 \
 nl_NL@euro/ISO-8859-15 \
 nn_NO.UTF-8/UTF-8 \
 nn_NO/ISO-8859-1 \
+oc_FR.UTF-8/UTF-8 \
 oc_FR/ISO-8859-1 \
 om_ET/UTF-8 \
+om_KE.UTF-8/UTF-8 \
 om_KE/ISO-8859-1 \
 pa_IN/UTF-8 \
 pl_PL.UTF-8/UTF-8 \
@@ -267,12 +279,17 @@ sk_SK.UTF-8/UTF-8 \
 sk_SK/ISO-8859-2 \
 sl_SI.UTF-8/UTF-8 \
 sl_SI/ISO-8859-2 \
+so_DJ.UTF-8/UTF-8 \
 so_DJ/ISO-8859-1 \
 so_ET/UTF-8 \
+so_KE.UTF-8/UTF-8 \
 so_KE/ISO-8859-1 \
+so_SO.UTF-8/UTF-8 \
 so_SO/ISO-8859-1 \
 sq_AL.UTF-8/UTF-8 \
 sq_AL/ISO-8859-1 \
+sr_CS.UTF-8/UTF-8 \
+sr_CS/ISO-8859-5 \
 st_ZA.UTF-8/UTF-8 \
 st_ZA/ISO-8859-1 \
 sv_FI.UTF-8/UTF-8 \
@@ -282,12 +299,14 @@ sv_SE.UTF-8/UTF-8 \
 sv_SE/ISO-8859-1 \
 ta_IN/UTF-8 \
 te_IN/UTF-8 \
+tg_TJ.UTF-8/UTF-8 \
 tg_TJ/KOI8-T \
 th_TH.UTF-8/UTF-8 \
 th_TH/TIS-620 \
 ti_ER/UTF-8 \
 ti_ET/UTF-8 \
 tig_ER/UTF-8 \
+tl_PH.UTF-8/UTF-8 \
 tl_PH/ISO-8859-1 \
 tr_TR.UTF-8/UTF-8 \
 tr_TR/ISO-8859-9 \
@@ -296,6 +315,7 @@ uk_UA.UTF-8/UTF-8 \
 uk_UA/KOI8-U \
 ur_PK/UTF-8 \
 uz_UZ/ISO-8859-1 \
+uz_UZ.UTF-8 UTF-8 \
 uz_UZ@cyrillic/UTF-8 \
 vi_VN.TCVN/TCVN5712-1 \
 vi_VN/UTF-8 \
@@ -304,6 +324,7 @@ wa_BE@euro/ISO-8859-15 \
 wa_BE.UTF-8/UTF-8 \
 xh_ZA.UTF-8/UTF-8 \
 xh_ZA/ISO-8859-1 \
+yi_US.UTF-8/UTF-8 \
 yi_US/CP1255 \
 zh_CN.GB18030/GB18030 \
 zh_CN.GBK/GBK \
@@ -311,6 +332,7 @@ zh_CN.UTF-8/UTF-8 \
 zh_CN/GB2312 \
 zh_HK.UTF-8/UTF-8 \
 zh_HK/BIG5-HKSCS \
+zh_SG.UTF-8/UTF-8 \
 zh_SG.GBK/GBK \
 zh_SG/GB2312 \
 zh_TW.EUC-TW/EUC-TW \
Index: localedata/locales/en_US
===================================================================
RCS file: /cvs/glibc/libc/localedata/locales/en_US,v
retrieving revision 1.9
retrieving revision 1.10
diff -u -p -r1.9 -r1.10
--- localedata/locales/en_US	31 Oct 2004 23:42:26 -0000	1.9
+++ localedata/locales/en_US	5 Jul 2005 18:35:41 -0000	1.10
@@ -100,7 +100,6 @@ mon	"<U004A><U0061><U006E><U0075><U0061>
 	"<U004F><U0063><U0074><U006F><U0062><U0065><U0072>";/
 	"<U004E><U006F><U0076><U0065><U006D><U0062><U0065><U0072>";/
 	"<U0044><U0065><U0063><U0065><U006D><U0062><U0065><U0072>"
-am_pm   "";""
 % Appropriate date and time representation (%c)
 %	"%a %d %b %Y %r %Z"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0072><U0020><U0025><U005A>"
Index: localedata/locales/fa_IR
===================================================================
RCS file: /cvs/glibc/libc/localedata/locales/fa_IR,v
retrieving revision 1.12
retrieving revision 1.13
diff -u -p -r1.12 -r1.13
--- localedata/locales/fa_IR	11 Oct 2004 22:27:10 -0000	1.12
+++ localedata/locales/fa_IR	27 Apr 2005 03:41:09 -0000	1.13
@@ -10,8 +10,8 @@ escape_char  /
 % Fax: +98 21 6019568
 % Language: fa
 % Territory: IR
-% Revision: 2.4
-% Date: 2004-09-04
+% Revision: 3.0
+% Date: 2005-04-06
 % Users: general
 % Repertoiremap:
 % Charset: UTF-8
@@ -28,28 +28,31 @@ tel        "+98 21 6022372"
 fax        "+98 21 6019568"
 language   "Persian"
 territory  "Iran"
-revision   "2.3"
-date       "2004-03-16"
+revision   "3.0"
+date       "2005-04-06"
 %
-category  "fa_IR:2004";LC_IDENTIFICATION
-category  "fa_IR:2004";LC_CTYPE
-category  "fa_IR:2004";LC_COLLATE
-category  "fa_IR:2004";LC_TIME
-category  "fa_IR:2004";LC_NUMERIC
-category  "fa_IR:2004";LC_MONETARY
-category  "fa_IR:2004";LC_MESSAGES
-category  "fa_IR:2004";LC_PAPER
-category  "fa_IR:2004";LC_NAME
-category  "fa_IR:2004";LC_ADDRESS
-category  "fa_IR:2004";LC_TELEPHONE
+category  "fa_IR:2005";LC_IDENTIFICATION
+category  "fa_IR:2005";LC_CTYPE
+category  "fa_IR:2005";LC_COLLATE
+category  "fa_IR:2005";LC_TIME
+category  "fa_IR:2005";LC_NUMERIC
+category  "fa_IR:2005";LC_MONETARY
+category  "fa_IR:2005";LC_MESSAGES
+category  "fa_IR:2005";LC_PAPER
+category  "fa_IR:2005";LC_NAME
+category  "fa_IR:2005";LC_ADDRESS
+category  "fa_IR:2005";LC_TELEPHONE
 
 END LC_IDENTIFICATION
 
 LC_CTYPE
 copy "i18n"
 
+% Persian uses the alternate digits U+06F0..U+06F9
 outdigit <U06F0>..<U06F9>
 
+% This is used in the scanf family of functions to read Persian numbers
+% using "%Id" and such.
 map to_inpunct; /
   (<U0030>,<U06F0>); /
   (<U0031>,<U06F1>); /
@@ -64,6 +67,8 @@ map to_inpunct; /
   (<U002E>,<U066B>); /
   (<U002C>,<U066C>)
 
+% This is used in the printf family of functions to write Persian floating
+% point numbers using "%If" and such.
 map to_outpunct; /
   (<U002E>,<U066B>); /
   (<U002C>,<U066C>)
@@ -76,7 +81,7 @@ copy "iso14651_t1"
 % kinds), BEH, PEH, TEH, JEEM, TCHEH, HAH, KHAH, DAL, THAL, REH, ZAIN, JEH,
 % SEEN, SHEEN, SAD, DAD, TAH, ZAH, AIN, GHAIN, FEH, QAF, KAF, GAF, LAM,
 % MEEM, NOON, WAW, HEH, YEH.
-% The various kind of HAMZA are sorted as ALEF WITH HAMZA ABOVE, ALEF WITH
+% The various kinds of HAMZA are sorted as ALEF WITH HAMZA ABOVE, ALEF WITH
 % HAMZA BELOW, WAW WITH HAMZA ABOVE, YEH WITH HAMZA ABOVE.
 
 collating-symbol <AHY> % accent hamza over yeh
@@ -122,7 +127,7 @@ reorder-after <waw>
 <heh>
 <yeh>
 
-% Persian uses fatha, kasra, damma, fathatan, kasratan, dammatan order.
+% Persian uses this order: Fatha, Kasra, Damma, Fathatan, Kasratan, Dammatan.
 
 reorder-after <U066D>
 <U064E> IGNORE;IGNORE;IGNORE;<U064E> %<fatha_no>
@@ -148,7 +153,7 @@ reorder-after <UFE7F>
 <U0655> IGNORE;IGNORE;IGNORE;<U0655> %<hamzabelow_no>
 <U0670> IGNORE;IGNORE;IGNORE;<U0670> %<supalef_no>
 
-% Persian digits are sorted before Arabic ones: they are the basic forms.
+% The Persian digits are sorted before the Arabic ones: they are the basic forms.
 reorder-after <U0660>
 <U06F0> <0>;<BAS>;<MIN>;IGNORE
 <U0660> <0>;<PCL>;<MIN>;IGNORE
@@ -292,14 +297,72 @@ grouping          3
 END LC_NUMERIC
 
 LC_TIME
-abday   "<U06CC><U002E>";"<U062F><U002E>";"<U0633><U002E>";/
-        "<U0686><U002E>";"<U067E><U002E>";"<U062C><U002E>";/
-        "<U0634><U002E>"
-day     "<U06CC><U06A9><U200C><U0634><U0646><U0628><U0647>";/
+% Alternative digits are used for Persian numerals in date and time. This is
+% a hack, until a new prefix is defined for alternative digits.
+alt_digits	"<U06F0><U06F0>";"<U06F0><U06F1>";/
+		"<U06F0><U06F2>";"<U06F0><U06F3>";/
+		"<U06F0><U06F4>";"<U06F0><U06F5>";/
+		"<U06F0><U06F6>";"<U06F0><U06F7>";/
+		"<U06F0><U06F8>";"<U06F0><U06F9>";/
+		"<U06F1><U06F0>";"<U06F1><U06F1>";/
+		"<U06F1><U06F2>";"<U06F1><U06F3>";/
+		"<U06F1><U06F4>";"<U06F1><U06F5>";/
+		"<U06F1><U06F6>";"<U06F1><U06F7>";/
+		"<U06F1><U06F8>";"<U06F1><U06F9>";/
+		"<U06F2><U06F0>";"<U06F2><U06F1>";/
+		"<U06F2><U06F2>";"<U06F2><U06F3>";/
+		"<U06F2><U06F4>";"<U06F2><U06F5>";/
+		"<U06F2><U06F6>";"<U06F2><U06F7>";/
+		"<U06F2><U06F8>";"<U06F2><U06F9>";/
+		"<U06F3><U06F0>";"<U06F3><U06F1>";/
+		"<U06F3><U06F2>";"<U06F3><U06F3>";/
+		"<U06F3><U06F4>";"<U06F3><U06F5>";/
+		"<U06F3><U06F6>";"<U06F3><U06F7>";/
+		"<U06F3><U06F8>";"<U06F3><U06F9>";/
+		"<U06F4><U06F0>";"<U06F4><U06F1>";/
+		"<U06F4><U06F2>";"<U06F4><U06F3>";/
+		"<U06F4><U06F4>";"<U06F4><U06F5>";/
+		"<U06F4><U06F6>";"<U06F4><U06F7>";/
+		"<U06F4><U06F8>";"<U06F4><U06F9>";/
+		"<U06F5><U06F0>";"<U06F5><U06F1>";/
+		"<U06F5><U06F2>";"<U06F5><U06F3>";/
+		"<U06F5><U06F4>";"<U06F5><U06F5>";/
+		"<U06F5><U06F6>";"<U06F5><U06F7>";/
+		"<U06F5><U06F8>";"<U06F5><U06F9>";/
+		"<U06F6><U06F0>";"<U06F6><U06F1>";/
+		"<U06F6><U06F2>";"<U06F6><U06F3>";/
+		"<U06F6><U06F4>";"<U06F6><U06F5>";/
+		"<U06F6><U06F6>";"<U06F6><U06F7>";/
+		"<U06F6><U06F8>";"<U06F6><U06F9>";/
+		"<U06F7><U06F0>";"<U06F7><U06F1>";/
+		"<U06F7><U06F2>";"<U06F7><U06F3>";/
+		"<U06F7><U06F4>";"<U06F7><U06F5>";/
+		"<U06F7><U06F6>";"<U06F7><U06F7>";/
+		"<U06F7><U06F8>";"<U06F7><U06F9>";/
+		"<U06F8><U06F0>";"<U06F8><U06F1>";/
+		"<U06F8><U06F2>";"<U06F8><U06F3>";/
+		"<U06F8><U06F4>";"<U06F8><U06F5>";/
+		"<U06F8><U06F6>";"<U06F8><U06F7>";/
+		"<U06F8><U06F8>";"<U06F8><U06F9>";/
+		"<U06F9><U06F0>";"<U06F9><U06F1>";/
+		"<U06F9><U06F2>";"<U06F9><U06F3>";/
+		"<U06F9><U06F4>";"<U06F9><U06F5>";/
+		"<U06F9><U06F6>";"<U06F9><U06F7>";/
+		"<U06F9><U06F8>";"<U06F9><U06F9>"
+% Persian doesn't have abbreviations for weekdays and month names, so
+% "abday" is the same as "day" and "abmon" is the same as "mon"
+abday   "<U06CC><U06A9><U0634><U0646><U0628><U0647>";/
         "<U062F><U0648><U0634><U0646><U0628><U0647>";/
         "<U0633><U0647><U200C><U0634><U0646><U0628><U0647>";/
         "<U0686><U0647><U0627><U0631><U0634><U0646><U0628><U0647>";/
-        "<U067E><U0646><U062C><U200C><U0634><U0646><U0628><U0647>";/
+        "<U067E><U0646><U062C><U0634><U0646><U0628><U0647>";/
+        "<U062C><U0645><U0639><U0647>";/
+        "<U0634><U0646><U0628><U0647>"
+day     "<U06CC><U06A9><U0634><U0646><U0628><U0647>";/
+        "<U062F><U0648><U0634><U0646><U0628><U0647>";/
+        "<U0633><U0647><U200C><U0634><U0646><U0628><U0647>";/
+        "<U0686><U0647><U0627><U0631><U0634><U0646><U0628><U0647>";/
+        "<U067E><U0646><U062C><U0634><U0646><U0628><U0647>";/
         "<U062C><U0645><U0639><U0647>";/
         "<U0634><U0646><U0628><U0647>"
 mon     "<U0698><U0627><U0646><U0648><U06CC><U0647>";/
@@ -314,33 +377,68 @@ mon     "<U0698><U0627><U0646><U0648><U0
         "<U0627><U0643><U062A><U0628><U0631>";/
         "<U0646><U0648><U0627><U0645><U0628><U0631>";/
         "<U062F><U0633><U0627><U0645><U0628><U0631>"
-abmon   "<U0698><U0627><U0646>";"<U0641><U0648><U0631>";/
-        "<U0645><U0627><U0631>";"<U0622><U0648><U0631>";/
-        "<U0645><U0640><U0647>";"<U0698><U0648><U0646>";/
-        "<U0698><U0648><U06CC>";"<U0627><U0648><U062A>";/
-        "<U0633><U067E><U062A>";"<U0627><U0643><U062A>";/
-        "<U0646><U0648><U0627>";"<U062F><U0633><U0627>"
-am_pm   "<U0635><U0628><U062D>";"<U0639><U0635><U0631>"
-d_t_fmt "<U202B><U0025><U0041><U0020><U0025><U0065><U0020><U0025>/
-<U0042><U0020><U0025><U0059><U060C><U0020><U0025><U0049><U003A>/
-<U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U202C>"
-d_fmt   "<U0025><U0059><U002F><U0025><U006D><U002F><U0025><U0064>"
-t_fmt   "<U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053>"
-t_fmt_ampm    "<U202B><U0025><U0049><U003A><U0025><U004D><U003A><U0025>/
-<U0053><U0020><U0025><U0070><U202C>"
-%date_fmt "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065><U0020>/
-%<U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025>/
-%<U005A><U0020><U0025><U0059>"
+abmon   "<U0698><U0627><U0646><U0648><U06CC><U0647>";/
+        "<U0641><U0648><U0631><U06CC><U0647>";/
+        "<U0645><U0627><U0631><U0633>";/
+        "<U0622><U0648><U0631><U06CC><U0644>";/
+        "<U0645><U0647>";/
+        "<U0698><U0648><U0626><U0646>";/
+        "<U0698><U0648><U0626><U06CC><U0647>";/
+        "<U0627><U0648><U062A>";/
+        "<U0633><U067E><U062A><U0627><U0645><U0628><U0631>";/
+        "<U0627><U0643><U062A><U0628><U0631>";/
+        "<U0646><U0648><U0627><U0645><U0628><U0631>";/
+        "<U062F><U0633><U0627><U0645><U0628><U0631>"
+% Persian does not have the 12-hour format
+am_pm   "";""
+t_fmt_ampm    ""
+%
+% Appropriate date representation (%x)
+%       "%Oy/%Om/%Od"
+d_fmt   "<U0025><U004F><U0079><U002F>/
+<U0025><U004F><U006D><U002F>/
+<U0025><U004F><U0064>"
+%
+% Appropriate time representation (%X)
+%       "%OH:%OM:%OS"
+t_fmt   "<U0025><U004F><U0048><U003A>/
+<U0025><U004F><U004D><U003A>/
+<U0025><U004F><U0053>"
+%
+% FIXME: need to add "HAMZA ABOVE" after January, February, May, July when used
+% before a year
+%
+% Appropriate date and time representation (%c)
+%       "<RLE>%A %Oe %B %Oy<ARABIC COMMA> %OH:%OM:%OS<PDF>"
+d_t_fmt "<U202B><U0025><U0041><U0020>/
+<U0025><U004F><U0065><U0020>/
+<U0025><U0042><U0020>/
+<U0025><U004F><U0079><U060C><U0020>/
+<U0025><U004F><U0048><U003A>/
+<U0025><U004F><U004D><U003A>/
+<U0025><U004F><U0053><U202C>"
+%
+% Appropriate date representation (date(1))
+%       "<RLE>%A %Oe %B %Oy<ARABIC COMMA> <SEEN><ALEF><AIN><TEH> %OH:%OM:%OS (%Z)<PDF>"
+date_fmt "<U202B><U0025><U0041><U0020>/
+<U0025><U004F><U0065><U0020>/
+<U0025><U0042><U0020>/
+<U0025><U004F><U0079><U060C><U0020>/
+<U0633><U0627><U0639><U062A><U0020>/
+<U0025><U004F><U0048><U003A>/
+<U0025><U004F><U004D><U003A>/
+<U0025><U004F><U0053><U0020>/
+<U0028><U0025><U005A><U0029><U202C>"
 first_weekday 7
 first_workday 7
 cal_direction 3
-%week    7;19971206;4
-%time_zone "???"
 END LC_TIME
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0079><U0059><U0628><U0066><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U062E><U0646><U006F><U005D><U002E><U002A>"
+% This is "^[yY<ALEF MADDA><BEH>Hf].*"
+yesexpr "<U005E><U005B><U0079><U0059><U0622><U0628><U0048><U0066><U005D><U002E><U002A>"
+% This is "^[nN<KHAH><NOON>ok].*"
+noexpr  "<U005E><U005B><U006E><U004E><U062E><U0646><U006F><U006B><U005D><U002E><U002A>"
 END LC_MESSAGES
 
 LC_PAPER
@@ -348,18 +446,6 @@ height   297
 width    210
 END LC_PAPER
 
-LC_TELEPHONE
-tel_int_fmt    "<U202A><U002B><U0025><U0063><U0020><U0025><U0061><U0020>/
-<U0025><U006C><U202C>"
-tel_dom_fmt    "<U202A><U0025><U0041><U2012><U0025><U006C><U202C>"
-int_select     "<U0030><U0030>"
-int_prefix     "<U0039><U0038>"
-END LC_TELEPHONE
-
-LC_MEASUREMENT
-measurement    1
-END LC_MEASUREMENT
-
 LC_NAME
 name_gen    ""
 name_miss   "<U062E><U0627><U0646><U0645>"
@@ -371,7 +457,6 @@ name_fmt    "<U0025><U0064><U0025><U0074
 END LC_NAME
 
 LC_ADDRESS
-% FIXME
 postal_fmt    "<U0025><U0066><U0025><U004E><U0025><U0061><U0025><U004E>/
 <U0025><U0064><U0025><U004E><U0025><U0062><U0025><U004E><U0025><U0073>/
 <U0020><U0025><U0068><U0020><U0025><U0065><U0020><U0025><U0072><U0025>/
@@ -388,3 +473,15 @@ lang_ab        "<U0066><U0061>"
 lang_term      "<U0066><U0061><U0073>"
 lang_lib       "<U0070><U0065><U0072>"
 END LC_ADDRESS
+
+LC_TELEPHONE
+tel_int_fmt    "<U202A><U002B><U0025><U0063><U0020><U0025><U0061><U0020>/
+<U0025><U006C><U202C>"
+tel_dom_fmt    "<U202A><U0025><U0041><U2012><U0025><U006C><U202C>"
+int_select     "<U0030><U0030>"
+int_prefix     "<U0039><U0038>"
+END LC_TELEPHONE
+
+LC_MEASUREMENT
+measurement    1
+END LC_MEASUREMENT
Index: localedata/locales/mn_MN
===================================================================
RCS file: /cvs/glibc/libc/localedata/locales/mn_MN,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -p -r1.4 -r1.5
--- localedata/locales/mn_MN	31 Oct 2004 23:42:27 -0000	1.4
+++ localedata/locales/mn_MN	22 May 2005 16:20:17 -0000	1.5
@@ -1,25 +1,27 @@
-comment_char %
 escape_char /
+comment_char %
 % Mongolian language (cyrillic) locale for Mongolia
 % Sanlig Badral <badral@chinggis.com>
-% Revision: 0.9 (2003-2-27)
+% Revision: 1.0 (2005-5-21)
 % Charsets: UTF-8
 % Distribution and use is free, also
 % for commercial purposes.
 
 LC_IDENTIFICATION
 title      "Mongolian locale for Mongolia"
-source     "OPENMN"
+source     "Sanlig Badral"
 address    ""
 contact    ""
-email      "bug-glibc-locales@gnu.org"
+email      "badral@chinggis.com"
 tel        ""
 fax        ""
 language   "Mongolian"
 territory  "Mongolia"
-revision   "0.9"
-date       "2003-02-27"
-
+revision   "1.0"
+date       "2005-05-21"
+audience   "general"
+application   "GNU locale"
+%
 category  "mn_MN:2000";LC_IDENTIFICATION
 category  "mn_MN:2000";LC_CTYPE
 category  "mn_MN:2000";LC_COLLATE
@@ -64,6 +66,82 @@ END LC_COLLATE
 
 LC_CTYPE
 copy "i18n"
+translit_start
+include "translit_combining";""
+% cyrillic -> latin for mongolian
+% CYRILLIC CAPITAL LETTERS IN ALPHABETICAL ORDER.
+<U0410> "<U0041>"
+<U0411> "<U0042>"
+<U0412> "<U0056>"
+<U0413> "<U0047>"
+<U0414> "<U0044>"
+<U0415> "<U0059><U0065>" % Ye
+<U0401> "<U0059><U006F>" % Yo
+<U0416> "<U004A>"
+<U0417> "<U005A>"
+<U0418> "<U0049>"
+<U0419> "<U0049>"
+<U041A> "<U004B>"
+<U041B> "<U004C>"
+<U041C> "<U004D>"
+<U041D> "<U004E>"
+<U041E> "<U004F>"
+<U04E8> "<U004F><U0065>" % Oe
+<U041F> "<U0050>"
+<U0420> "<U0052>"
+<U0421> "<U0053>"
+<U0422> "<U0054>"
+<U0423> "<U0055>"
+<U04AE> "<U0055><U0065>" % Ue
+<U0424> "<U0046>"
+<U0425> "<U0048>"
+<U0426> "<U0043>"
+<U0427> "<U0043><U0068>" % Ch
+<U0428> "<U0053><U0068>" % Sh
+<U0429> "<U0053><U0068>" % Sh
+<U042A> "<U0022>" % "
+<U042B> "<U0059>"
+<U042C> "<U0027>" % '
+<U042D> "<U0045>" 
+<U042E> "<U0059><U0075>" % Yu
+<U042F> "<U0059><U0061>" % Ya
+% CYRILLIC SMALL LETTERS IN ALPHABETICAL ORDER.
+<U0430> "<U0061>"
+<U0431> "<U0062>"
+<U0432> "<U0076>"
+<U0433> "<U0067>"
+<U0434> "<U0064>"
+<U0435> "<U0079><U0065>" %ye
+<U0451> "<U0079><U006F>" %yo
+<U0436> "<U006A>"
+<U0437> "<U007A>"
+<U0438> "<U0069>"
+<U0439> "<U0069>"
+<U043A> "<U006B>"
+<U043B> "<U006C>"
+<U043C> "<U006D>"
+<U043D> "<U006E>"
+<U043E> "<U006F>"
+<U04E9> "<U006F><U0065>" % oe
+<U043F> "<U0070>"
+<U0440> "<U0072>"
+<U0441> "<U0073>"
+<U0442> "<U0074>"
+<U0443> "<U0075>"
+<U04AF> "<U0075><U0065>" % ue
+<U0444> "<U0066>"
+<U0445> "<U0068>"
+<U0446> "<U0043>"
+<U0447> "<U0063><U0068>" % ch
+<U0448> "<U0073><U0068>" % sh
+<U0449> "<U0073><U0068>" % sh
+<U044A> "<U0022>" % "
+<U044B> "<U0079>"
+<U044C> "<U0027>" % '
+<U044D> "<U0065>"
+<U044E> "<U0079><U0075>" % yu
+<U044F> "<U0079><U0061>" % ya
+translit_end
 END LC_CTYPE
 
 LC_MONETARY
@@ -106,28 +184,28 @@ day   "<U041D><U044F><U043C>";/
       "<U0411><U0430><U0430><U0441><U0430><U043D>";/
       "<U0411><U044F><U043C><U0431><U0430>"
 % Abbreviated month names (%b)
-abmon       "<U0031><U002D><U0440>";"<U0032><U002D><U0440>";/
-            "<U0033><U002D><U0440>";"<U0034><U002D><U0440>";/
-            "<U0035><U002D><U0440>";"<U0036><U002D><U0440>";/
-            "<U0037><U002D><U0440>";"<U0038><U002D><U0440>";/
-            "<U0039><U002D><U0440>";"<U0031><U0030><U002D><U0440>";/
-            "<U0031><U0031><U002D><U0440>";"<U0031><U0032><U002D><U0440>"
+abmon       "<U0425><U0443><U043B>";"<U04AE><U0445><U044D>";/
+            "<U0411><U0430><U0440>";"<U0422><U0443><U0443>";/
+            "<U041B><U0443><U0443>";"<U041C><U043E><U0433>";/
+            "<U041C><U043E><U0440>";"<U0425><U043E><U043D>";/
+            "<U0411><U0438><U0447>";"<U0422><U0430><U0445>";/
+            "<U041D><U043E><U0445>";"<U0413><U0430><U0445>"
 % Full month names (%B)
-mon         "<U041D><U044D><U0433><U0434><U04AF><U0433><U044D><U044D><U0440><U0020><U0441><U0430><U0440>";/
-            "<U0425><U043E><U0451><U0440><U0434><U0443><U0433><U0430><U0430><U0440><U0020><U0441><U0430><U0440>";/
-            "<U0413><U0443><U0440><U0430><U0432><U0434><U0443><U0433><U0430><U0430><U0440><U0020><U0441><U0430><U0440>";/
-            "<U0414><U04E9><U0440><U04E9><U0432><U0434><U04AF><U0433><U044D><U044D><U0440><U0020><U0441><U0430><U0440>";/
-            "<U0422><U0430><U0432><U0434><U0443><U0433><U0430><U0430><U0440><U0020><U0441><U0430><U0440>";/
-            "<U0417><U0443><U0440><U0433><U0430><U0430><U0434><U0443><U0433><U0430><U0440><U0020><U0441><U0430><U0440>";/
-            "<U0414><U043E><U043B><U043E><U043E><U0434><U0443><U0433><U0430><U0430><U0440><U0020><U0441><U0430><U0440>";/
-            "<U041D><U0430><U0439><U043C><U0434><U0443><U0433><U0430><U0430><U0440><U0020><U0441><U0430><U0440>";/
-            "<U0415><U0441><U0434><U04AF><U0433><U044D><U044D><U0440><U0020><U0441><U0430><U0440>";/
-            "<U0410><U0440><U0430><U0432><U0434><U0443><U0433><U0430><U0430><U0440><U0020><U0441><U0430><U0440>";/
-            "<U0410><U0440><U0432><U0430><U043D><U043D><U044D><U0433><U0434><U04AF><U0433><U044D><U044D><U0440><U0020><U0441><U0430><U0440>";/
-            "<U0410><U0440><U0432><U0430><U043D><U0445><U043E><U0451><U0440><U0434><U0433><U0430><U0430><U0440><U0020><U0441><U0430><U0440>"
+mon         "<U0425><U0443><U043B><U0433><U0430><U043D><U0430><U0020><U0441><U0430><U0440><U044B><U043D>";/
+            "<U04AE><U0445><U044D><U0440><U0020><U0441><U0430><U0440><U044B><U043D>";/
+            "<U0411><U0430><U0440><U0020><U0441><U0430><U0440><U044B><U043D>";/
+            "<U0422><U0443><U0443><U043B><U0430><U0439><U0020><U0441><U0430><U0440><U044B><U043D>";/
+            "<U041B><U0443><U0443><U0020><U0441><U0430><U0440><U044B><U043D>";/
+            "<U041C><U043E><U0433><U043E><U0439><U0020><U0441><U0430><U0440><U044B><U043D>";/
+            "<U041C><U043E><U0440><U044C><U0020><U0441><U0430><U0440><U044B><U043D>";/
+            "<U0425><U043E><U043D><U044C><U0020><U0441><U0430><U0440><U044B><U043D>";/
+            "<U0411><U0438><U0447><U0020><U0441><U0430><U0440><U044B><U043D>";/
+            "<U0422><U0430><U0445><U0438><U0430><U0020><U0441><U0430><U0440><U044B><U043D>";/
+            "<U041D><U043E><U0445><U043E><U0439><U0020><U0441><U0430><U0440><U044B><U043D>";/
+            "<U0413><U0430><U0445><U0430><U0439><U0020><U0441><U0430><U0440><U044B><U043D>"
 % Appropriate date and time representation
-% "%a %d %b %Y %T %Z"
-d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
+% "%Y %b %d, %a %T"
+d_t_fmt "<U0025><U0059><U0020><U0025><U0062><U0020><U0025><U0064><U002C><U0020><U0025><U0061><U0020><U0025><U0054>"
 
 % Appropriate date representation
 % "%Y.%m.%d"
@@ -139,8 +217,9 @@ t_fmt   "<U0025><U0054>"
 am_pm   "";""
 t_fmt_ampm ""
 % Full date and time representation
-% "%a %b %e %H:%M:%S %Z %Y"
-date_fmt  "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065><U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A><U0020><U0025><U0059>"
+% "%Z %Y ony %B %e, %a %H:%M:%S"
+date_fmt  "<U0025><U005A><U0020><U0025><U0059><U0020><U043E><U043D><U044B><U0020><U0025><U0042><U0020>";/
+	  "<U0025><U0065><U002C><U0020><U0025><U0061><U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053>"
 END LC_TIME
 
 LC_MESSAGES
@@ -157,7 +236,6 @@ END LC_PAPER
 LC_NAME
 name_fmt    "<U0025><U0064><U0025><U0074><U0025><U0067><U0025><U0074>/
 <U0025><U006D><U0025><U0074><U0025><U0066>"
-
 name_miss   "<U0425><U0430><U0442><U0430><U0433><U0442><U0430><U0439>"
 name_mr     "<U041D><U043E><U0451><U043D>"
 name_mrs    "<U0425><U0430><U0442><U0430><U0433><U0442><U0430><U0439>"
@@ -170,7 +248,6 @@ postal_fmt    "<U0025><U0066><U0025><U00
 <U0020><U0025><U0068><U0020><U0025><U0065><U0020><U0025><U0072><U0025>/
 <U004E><U0025><U0025><U007A><U0020><U0025><U0054><U0025>/
 <U004E><U0025><U0063><U0025><U004E>"
-
 country_name  "<U004D><U006F><U006E><U0067><U006F><U006C><U0069><U0061>"
 country_post  "<U004D><U004E>"
 country_ab2   "<U004D><U004E>"
Index: localedata/locales/pa_IN
===================================================================
RCS file: /cvs/glibc/libc/localedata/locales/pa_IN,v
retrieving revision 1.6
retrieving revision 1.8
diff -u -p -r1.6 -r1.8
--- localedata/locales/pa_IN	27 Nov 2004 19:08:24 -0000	1.6
+++ localedata/locales/pa_IN	8 Jun 2005 20:22:49 -0000	1.8
@@ -5,7 +5,7 @@ escape_char     /
 
 LC_IDENTIFICATION
 title      "Punjabi language locale for Indian Punjabi(Gurmukhi)"
-source     "IndLinux.org"	
+source     "IndLinux.org"
 address    ""
 contact    ""
 email      "bug-glibc-locales@gnu.org"
@@ -14,7 +14,7 @@ fax        ""
 language   "Punjabi"
 territory  "India"
 revision   "0.2"
-date       "2004-09-30"	
+date       "2004-09-30"
 %
 category  "pa_IN:2000";LC_IDENTIFICATION
 category  "pa_IN:2000";LC_CTYPE
@@ -122,8 +122,8 @@ mon         "<U0A1C><U0A28><U0A35><U0A30
 
 %
 % Equivalent of AM PM
-am_pm       "<U0A38><U035><U0A47><U0A30><U0A47>";/
-            "<U0A36><U0A3E><U02E>"
+am_pm       "<U0A38><U0A35><U0A47><U0A30><U0A47>";/
+            "<U0A36><U0A3E><U0A2E>"
 
 %
 % Appropriate date and time representation
Index: localedata/locales/vi_VN
===================================================================
RCS file: /cvs/glibc/libc/localedata/locales/vi_VN,v
retrieving revision 1.6
retrieving revision 1.7
diff -u -p -r1.6 -r1.7
--- localedata/locales/vi_VN	31 Oct 2004 23:42:28 -0000	1.6
+++ localedata/locales/vi_VN	8 Jul 2005 16:19:07 -0000	1.7
@@ -1,23 +1,35 @@
-comment_char    %
-escape_char     /
-% Vietnamese language locale for Vietnam.
-% Contributed by Kentaroh Noji <knoji@jp.ibm.com> and
-% Tetsuji Orita <orita@jp.ibm.com>.
-
-% Revised by Le Hong Boi
+escape_char /
+comment_char %
+%
+% Vietnamese language locale for Vietnam
+% Source: TCVN table and info from vietnamese native speakers
+% Address:
+% Contact: Pablo Saratxaga
+% Email: srtxg@chanae.alphanet.ch
+% Tel:
+% Fax:
+% Language: vi
+% Territory: VN
+% Revision: 0.7
+% Date: 2003-02-06
+% Users: general
+% Repertoiremap:
+% Charset: UTF-8
+% Distribution and use is free, also
+% for commercial purposes.
 
 LC_IDENTIFICATION
 title      "Vietnamese language locale for Vietnam"
-source     "IBM Globalization Center of Competency, Yamato Software Laboratory"
-address    "1623-14, Shimotsuruma, Yamato-shi, Kanagawa-ken, 242-8502, Japan"
-contact    ""
-email      "bug-glibc-locales@gnu.org"
+source     "TCVN table and info from vietnamese native speakers"
+address    ""
+contact    "Pablo Saratxaga"
+email      "srtxg@chanae.alphanet.ch"
 tel        ""
 fax        ""
 language   "Vietnamese"
 territory  "Vietnam"
-revision   "1.0"
-date       "2000-07-20"
+revision   "0.6"
+date       "2001-01-29"
 %
 category  "vi_VN:2000";LC_IDENTIFICATION
 category  "vi_VN:2000";LC_CTYPE
@@ -30,200 +42,248 @@ category  "vi_VN:2000";LC_PAPER
 category  "vi_VN:2000";LC_NAME
 category  "vi_VN:2000";LC_ADDRESS
 category  "vi_VN:2000";LC_TELEPHONE
+category  "vi_VN:2000";LC_MEASUREMENT
 
 END LC_IDENTIFICATION
 
 LC_CTYPE
 copy "i18n"
+translit_start
+
+% dong sign -> d// -> dd
+<U20AB> "<U0111>";"<U0064><U0064>"
+
+translit_end
 END LC_CTYPE
 
 LC_COLLATE
-
 % Copy the template from ISO/IEC 14651
 copy "iso14651_t1"
 
+% A( A^ E^ O+ U+ DD are treatead as base letters
+% ordering of accents is: grave, hook, tilde, acute, dot below
+% a a( a> b c d d/ e e> f g h i j k l m n o o+ p q r s t u u+ v w x y z
+
+collating-symbol <a(>
+collating-symbol <a/>>
+collating-symbol <d//>
+collating-symbol <e/>>
+collating-symbol <o/>>
+collating-symbol <o9>
+collating-symbol <u9>
+
+collating-symbol <HOK>
+
+reorder-after <BAS>
+<GRA>
+<HOK>
+<TIL>
+<ACA>
+<PCT>
+
+reorder-after <a>
+<a(>
+<a/>>
+reorder-after <d>
+<d//>
+reorder-after <e>
+<e/>>
+reorder-after <o>
+<o/>>
+<o9>
+reorder-after <u>
+<u9>
+
+reorder-after <U0061>
+<U1EA3> <a>;<HOK>;<MIN>;IGNORE
+<U0103> <a(>;<BAS>;<MIN>;IGNORE
+<U1EB1> <a(>;<GRA>;<MIN>;IGNORE
+<U1EB3> <a(>;<HOK>;<MIN>;IGNORE
+<U1EB5> <a(>;<TIL>;<MIN>;IGNORE
+<U1EAF> <a(>;<ACA>;<MIN>;IGNORE
+<U1EB7> <a(>;<PCT>;<MIN>;IGNORE
+<U00E2> <a/>>;<BAS>;<MIN>;IGNORE
+<U1EA7> <a/>>;<GRA>;<MIN>;IGNORE
+<U1EA9> <a/>>;<HOK>;<MIN>;IGNORE
+<U1EAB> <a/>>;<TIL>;<MIN>;IGNORE
+<U1EA5> <a/>>;<ACA>;<MIN>;IGNORE
+<U1EAD> <a/>>;<PCT>;<MIN>;IGNORE
+reorder-after <U0041>
+<U1EA2> <a>;<HOK>;<CAP>;IGNORE
+<U0102> <a(>;<BAS>;<CAP>;IGNORE
+<U1EB0> <a(>;<GRA>;<CAP>;IGNORE
+<U1EB3> <a(>;<HOK>;<CAP>;IGNORE
+<U1EB4> <a(>;<TIL>;<CAP>;IGNORE
+<U1EAE> <a(>;<ACA>;<CAP>;IGNORE
+<U1EB6> <a(>;<PCT>;<CAP>;IGNORE
+<U00C2> <a/>>;<BAS>;<CAP>;IGNORE
+<U1EA6> <a/>>;<GRA>;<CAP>;IGNORE
+<U1EA8> <a/>>;<HOK>;<CAP>;IGNORE
+<U1EAA> <a/>>;<TIL>;<CAP>;IGNORE
+<U1EA4> <a/>>;<ACA>;<CAP>;IGNORE
+<U1EAC> <a/>>;<PCT>;<CAP>;IGNORE
+
+reorder-after <U0064>
+<U0111> <d//>;<BAS>;<MIN>;IGNORE
+<U00F0> <d//>;<PCL>;<MIN>;IGNORE
+reorder-after <U0044>
+<U0110> <d//>;<BAS>;<CAP>;IGNORE
+<U00D0> <d//>;<PCL>;<CAP>;IGNORE
+
+reorder-after <U0065>
+<U1EBB> <e>;<HOK>;<MIN>;IGNORE
+<U00EA> <e/>>;<BAS>;<MIN>;IGNORE
+<U1EC1> <e/>>;<GRA>;<MIN>;IGNORE
+<U1EC3> <e/>>;<HOK>;<MIN>;IGNORE
+<U1EC5> <e/>>;<TIL>;<MIN>;IGNORE
+<U1EBF> <e/>>;<ACA>;<MIN>;IGNORE
+<U1EC7> <e/>>;<PCT>;<MIN>;IGNORE
+reorder-after <U0045>
+<U1EBA> <e>;<HOK>;<CAP>;IGNORE
+<U00CA> <e/>>;<BAS>;<CAP>;IGNORE
+<U1EC0> <e/>>;<GRA>;<CAP>;IGNORE
+<U1EC2> <e/>>;<HOK>;<CAP>;IGNORE
+<U1EC4> <e/>>;<TIL>;<CAP>;IGNORE
+<U1EBE> <e/>>;<ACA>;<CAP>;IGNORE
+<U1EC6> <e/>>;<PCT>;<CAP>;IGNORE
+
+reorder-after <U0069>
+<U1EC9> <i>;<HOK>;<MIN>;IGNORE
+reorder-after <U0049>
+<U1EC8> <i>;<HOK>;<CAP>;IGNORE
+
+reorder-after <U006F>
+<U1ECF> <o>;<HOK>;<MIN>;IGNORE
+<U00F4> <o/>>;<BAS>;<MIN>;IGNORE
+<U1ED3> <o/>>;<GRA>;<MIN>;IGNORE
+<U1ED5> <o/>>;<HOK>;<MIN>;IGNORE
+<U1ED7> <o/>>;<TIL>;<MIN>;IGNORE
+<U1ED1> <o/>>;<ACA>;<MIN>;IGNORE
+<U1ED9> <o/>>;<PCT>;<MIN>;IGNORE
+<U01A1> <o9>;<BAS>;<MIN>;IGNORE
+<U1EDD> <o9>;<GRA>;<MIN>;IGNORE
+<U1EDF> <o9>;<HOK>;<MIN>;IGNORE
+<U1EE1> <o9>;<TIL>;<MIN>;IGNORE
+<U1EDB> <o9>;<ACA>;<MIN>;IGNORE
+<U1EE3> <o9>;<PCT>;<MIN>;IGNORE
+reorder-after <U004F>
+<U1ECE> <o>;<HOK>;<CAP>;IGNORE
+<U00D4> <o/>>;<BAS>;<CAP>;IGNORE
+<U1ED2> <o/>>;<GRA>;<CAP>;IGNORE
+<U1ED4> <o/>>;<HOK>;<CAP>;IGNORE
+<U1ED6> <o/>>;<TIL>;<CAP>;IGNORE
+<U1ED0> <o/>>;<ACA>;<CAP>;IGNORE
+<U1ED8> <o/>>;<PCT>;<CAP>;IGNORE
+<U01A0> <o9>;<BAS>;<CAP>;IGNORE
+<U1EDC> <o9>;<GRA>;<CAP>;IGNORE
+<U1EDE> <o9>;<HOK>;<CAP>;IGNORE
+<U1EE0> <o9>;<TIL>;<CAP>;IGNORE
+<U1EDA> <o9>;<ACA>;<CAP>;IGNORE
+<U1EE2> <o9>;<PCT>;<CAP>;IGNORE
+
+reorder-after <U0075>
+<U1EE7> <u>;<HOK>;<MIN>;IGNORE
+<U01B0> <u9>;<BAS>;<MIN>;IGNORE
+<U1EEB> <u9>;<GRA>;<MIN>;IGNORE
+<U1EED> <u9>;<HOK>;<MIN>;IGNORE
+<U1EEF> <u9>;<TIL>;<MIN>;IGNORE
+<U1EE9> <u9>;<ACA>;<MIN>;IGNORE
+<U1EF1> <u9>;<PCT>;<MIN>;IGNORE
+reorder-after <U0055>
+<U1EE6> <u>;<HOK>;<CAP>;IGNORE
+<U01AF> <u9>;<BAS>;<CAP>;IGNORE
+<U1EEA> <u9>;<GRA>;<CAP>;IGNORE
+<U1EEC> <u9>;<HOK>;<CAP>;IGNORE
+<U1EEE> <u9>;<TIL>;<CAP>;IGNORE
+<U1EE8> <u9>;<ACA>;<CAP>;IGNORE
+<U1EF0> <u9>;<PCT>;<CAP>;IGNORE
+
+reorder-after <U0079>
+<U1EF7> <y>;<HOK>;<MIN>;IGNORE
+reorder-after <U0059>
+<U1EF6> <y>;<HOK>;<CAP>;IGNORE
+
+reorder-end
+
 END LC_COLLATE
 
 LC_MONETARY
-% This is the POSIX Locale definition the LC_MONETARY category.
-% These are generated based on XML base Locale difintion file
-% for IBM Class for Unicode/Java
-%
 int_curr_symbol       "<U0056><U004E><U0044><U0020>"
 currency_symbol       "<U20AB>"
 mon_decimal_point     "<U002C>"
 mon_thousands_sep     "<U002E>"
-mon_grouping          3
+mon_grouping          3;3
 positive_sign         ""
 negative_sign         "<U002D>"
-int_frac_digits       2
-frac_digits           2
+int_frac_digits       4
+frac_digits           4
 p_cs_precedes         0
 p_sep_by_space        0
 n_cs_precedes         1
 n_sep_by_space        0
 p_sign_posn           1
 n_sign_posn           1
-%
 END LC_MONETARY
 
-
 LC_NUMERIC
-% This is the POSIX Locale definition for the LC_NUMERIC  category.
-%
-decimal_point          "<U002C>"
-thousands_sep          "<U002E>"
-grouping               3
-%
+decimal_point        "<U002C>"
+thousands_sep        "<U002E>"
+grouping             3;3
 END LC_NUMERIC
 
-
 LC_TIME
-% This is the POSIX Locale definition for the LC_TIME category.
-% These are generated based on XML base Locale difintion file
-% for IBM Class for Unicode/Java
-%
-% Abbreviated weekday names (%a)
-abday       "<U0043><U004E><U0020>";/
-            "<U0054><U0068><U0020><U0032><U0020>";/
-            "<U0054><U0068><U0020><U0033><U0020>";/
-            "<U0054><U0068><U0020><U0034><U0020>";/
-            "<U0054><U0068><U0020><U0035><U0020>";/
-            "<U0054><U0068><U0020><U0036><U0020>";/
-            "<U0054><U0068><U0020><U0037><U0020>"
-%
-% Full weekday names (%A)
-day         "<U0043><U0068><U0075><U0309><U0020><U006E><U0068><U00E2><U0323>/
-<U0074><U0020>";/
-            "<U0054><U0068><U01B0><U0301><U0020><U0068><U0061><U0069><U0020>";/
-            "<U0054><U0068><U01B0><U0301><U0020><U0062><U0061><U0020>";/
-            "<U0054><U0068><U01B0><U0301><U0020><U0074><U01B0><U0020>";/
-            "<U0054><U0068><U01B0><U0301><U0020><U006E><U0103><U006D><U0020>";/
-            "<U0054><U0068><U01B0><U0301><U0020><U0073><U00E1><U0075><U0020>";/
-            "<U0054><U0068><U01B0><U0301><U0020><U0062><U0061><U0309><U0079>/
-<U0020>"
-%
-% Abbreviated month names (%b)
-abmon       "<U0054><U0068><U0067><U0020><U0031>";/
-            "<U0054><U0068><U0067><U0020><U0032>";/
-            "<U0054><U0068><U0067><U0020><U0033>";/
-            "<U0054><U0068><U0067><U0020><U0034>";/
-            "<U0054><U0068><U0067><U0020><U0035>";/
-            "<U0054><U0068><U0067><U0020><U0036>";/
-            "<U0054><U0068><U0067><U0020><U0037>";/
-            "<U0054><U0068><U0067><U0020><U0038>";/
-            "<U0054><U0068><U0067><U0020><U0039>";/
-            "<U0054><U0068><U0067><U0020><U0031><U0030>";/
-            "<U0054><U0068><U0067><U0020><U0031><U0031>";/
-            "<U0054><U0068><U0067><U0020><U0031><U0032>"
-%
-% Full month names (%B)
-mon         "<U0054><U0068><U00E1><U006E><U0067><U0020><U006D><U00F4><U0323>/
-<U0074>";/
-            "<U0054><U0068><U00E1><U006E><U0067><U0020><U0068><U0061><U0069>";/
-            "<U0054><U0068><U00E1><U006E><U0067><U0020><U0062><U0061>";/
-            "<U0054><U0068><U00E1><U006E><U0067><U0020><U0074><U01B0>";/
-            "<U0054><U0068><U00E1><U006E><U0067><U0020><U006E><U0103><U006D>";/
-            "<U0054><U0068><U00E1><U006E><U0067><U0020><U0073><U00E1><U0075>";/
-            "<U0054><U0068><U00E1><U006E><U0067><U0020><U0062><U0061><U0309>/
-<U0079>";/
-            "<U0054><U0068><U00E1><U006E><U0067><U0020><U0074><U00E1><U006D>";/
-            "<U0054><U0068><U00E1><U006E><U0067><U0020><U0063><U0068><U00ED>/
-<U006E>";/
-            "<U0054><U0068><U00E1><U006E><U0067><U0020><U006D><U01B0><U01A1>/
-<U0300><U0069>";/
-            "<U0054><U0068><U00E1><U006E><U0067><U0020><U006D><U01B0><U01A1>/
-<U0300><U0069><U0020><U006D><U00F4><U0323><U0074>";/
-            "<U0054><U0068><U00E1><U006E><U0067><U0020><U006D><U01B0><U01A1>/
-<U0300><U0069><U0020><U0068><U0061><U0069>"
-%
-% Equivalent of AM PM
-am_pm       "";""
-%
-% Appropriate date and time representation
-% %A %d %b %Y%H:%M:%S %Z
-d_t_fmt     "<U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053>/
-<U0020><U0025><U005A><U0020><U0025><U0041><U0020><U0025><U0064><U0020>/
-<U0025><U0062><U0020><U0025><U0059>"
-%
-% Appropriate date representation
-% %A %d %b %Y
-d_fmt       "<U0025><U0041><U0020><U0025><U0064><U0020><U0025><U0062>/
-<U0020><U0025><U0059>"
-%
-% Appropriate time representation
-% %H:%M:%S %Z
-t_fmt       "<U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053>/
-<U0020><U0025><U005A>"
-%
-% Appropriate 12 h time representation (%r)
-t_fmt_ampm ""
-%
-% Appropriate date representation (date(1))   "%H:%M:%S %a %e %b %Y %Z"
-date_fmt       "<U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053>/
-<U0020><U0025><U0061><U0020><U0025><U0065><U0020><U0025><U0062>/
-<U0020><U0025><U0059><U0020><U0025><U005A>"
+abday    "<U0043><U004E>";"<U0054><U0068><U0032>";/
+         "<U0054><U0068><U0033>";"<U0054><U0068><U0034>";/
+         "<U0054><U0068><U0035>";"<U0054><U0068><U0036>";/
+         "<U0054><U0068><U0037>"
+day      "<U0063><U0068><U1EE7><U0020><U006E><U0068><U1EAD><U0074>";/
+         "<U0074><U0068><U1EE9><U0020><U0068><U0061><U0069>";/
+         "<U0074><U0068><U1EE9><U0020><U0062><U0061>";/
+         "<U0074><U0068><U1EE9><U0020><U0074><U01B0>";/
+         "<U0074><U0068><U1EE9><U0020><U006E><U0103><U006D>";/
+         "<U0074><U0068><U1EE9><U0020><U0073><U00E1><U0075>";/
+         "<U0074><U0068><U1EE9><U0020><U0062><U1EA3><U0079>"
+abmon    "<U0054><U0068><U0067><U0031>";"<U0054><U0068><U0067><U0032>";/
+         "<U0054><U0068><U0067><U0033>";"<U0054><U0068><U0067><U0034>";/
+         "<U0054><U0068><U0067><U0035>";"<U0054><U0068><U0067><U0036>";/
+         "<U0054><U0068><U0067><U0037>";"<U0054><U0068><U0067><U0038>";/
+         "<U0054><U0068><U0067><U0039>";"<U0054><U0068><U0067><U0031><U0030>";/
+         "<U0054><U0068><U0067><U0031><U0031>";"<U0054><U0068><U0067><U0031><U0032>"
+mon      "<U0074><U0068><U00E1><U006E><U0067><U0020><U0067><U0069><U00EA><U006E><U0067>";/
+         "<U0074><U0068><U00E1><U006E><U0067><U0020><U0068><U0061><U0069>";/
+         "<U0074><U0068><U00E1><U006E><U0067><U0020><U0062><U0061>";/
+         "<U0074><U0068><U00E1><U006E><U0067><U0020><U0074><U01B0>";/
+         "<U0074><U0068><U00E1><U006E><U0067><U0020><U006E><U0103><U006D>";/
+         "<U0074><U0068><U00E1><U006E><U0067><U0020><U0073><U00E1><U0075>";/
+         "<U0074><U0068><U00E1><U006E><U0067><U0020><U0062><U1EA3><U0079>";/
+         "<U0074><U0068><U00E1><U006E><U0067><U0020><U0074><U00E1><U006D>";/
+         "<U0074><U0068><U00E1><U006E><U0067><U0020><U0063><U0068><U00ED><U006E>";/
+         "<U0074><U0068><U00E1><U006E><U0067><U0020><U006D><U01B0><U1EDD><U0069>";/
+         "<U0074><U0068><U00E1><U006E><U0067><U0020><U006D><U1ED9><U0074>";/
+         "<U0074><U0068><U00E1><U006E><U0067><U0020><U0063><U0068><U1EA1><U0070>"
+d_t_fmt  "<U0025><U0041><U002C><U0020><U0025><U0064><U0020><U0025><U0042><U0020><U006E><U0103><U006D><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
+d_fmt    "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0059>"
+t_fmt    "<U0025><U0054>"
+am_pm    "<U0073><U00E1><U006E><U0067>";"<U0063><U0068><U0069><U1EC1><U0075>"
+t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U0020><U0025><U0070>"
+date_fmt  "<U0025><U0041><U002C><U0020><U0025><U0064><U0020><U0025><U0042><U0020><U006E><U0103><U006D><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 END LC_TIME
 
-
+% yes: Va^ng / Da.
+% no: Kho^ng
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+yesstr   "<U0056><U00E2><U006E><U0067>"
+nostr    "<U004B><U0068><U00F4><U006E><U0067>"
+yesexpr  "<U005E><U005B><U0031><U0079><U0059><U0076><U0056><U0064><U0044><U0063><U0043><U005D><U002E><U002A>"
+noexpr   "<U005E><U005B><U0030><U006E><U004E><U006B><U004B><U005D><U002E><U002A>"
 END LC_MESSAGES
 
-
 LC_PAPER
-% This is the ISO_IEC TR14652 Locale definition for the
-% LC_PAPER category
 height      297
 width       210
-
 END LC_PAPER
 
-
-LC_NAME
-% This is the ISO_IEC TR14652 Locale definition for the
-% LC_NAME category.
-%
-name_fmt    "<U0025><U0070><U0025><U0074><U0025><U0066><U0025><U0074>/
-<U0025><U0067>"
-name_gen    ""
-name_mr     "<U00D4><U002E>"
-name_mrs    "<U0042><U002E>"
-name_miss   "<U0043><U002E>"
-name_ms     ""
-
-END LC_NAME
-
-
-LC_ADDRESS
-% This is the ISO_IEC TR14652 Locale definition for the
-% LC_ADDRESS
-postal_fmt  "<U0025><U007A><U0025><U0063><U0025><U0054><U0025><U0073>/
-<U0025><U0062><U0025><U0065><U0025><U0072>"
-country_name "<U0056><U0069><U1EC7><U0074><U0020><U006E><U0061><U006D>"
-lang_name "<U0056><U0069><U1EC7><U0074><U0020><U006E><U0067><U1EEF>"
-
-country_ab2 "<U0056><U004E>"
-country_ab3 "<U0056><U004E><U004D>"
-country_num 704
-END LC_ADDRESS
-
-
-LC_TELEPHONE
-% This is the ISO_IEC TR14652 Locale definition for the
-%
-tel_int_fmt "<U002B><U0025><U0063><U0020><U003B><U0025><U0061><U0020>/
-<U003B><U0025><U006C>"
-int_prefix  "<U0038><U0034>"
-int_select  "<U0030><U0030>"
-
-END LC_TELEPHONE
-
-
 LC_MEASUREMENT
-% This is the ISO_IEC TR14652 Locale definition for the
-%
 measurement 1
-
 END LC_MEASUREMENT
+
--- /dev/null	2005-04-05 09:45:02.246335296 +0900
+++ localedata/locales/sr_CS	2005-03-20 13:16:06.000000000 +0900
@@ -0,0 +1,349 @@
+comment_char %
+escape_char  /
+%
+% Serbian Language Locale for Serbia and Montenegro
+% Source: sr_YU locale
+% Address:
+%
+% Contact: Danilo Segan
+% Email: dsegan@gmx.net
+% Tel: +381 63 8626082
+% Fax: none
+% Language: sr
+% Territory: CS
+% Revision: 1.2
+% Date: 2004-08-09
+% Application: general
+% Users: general
+% Charset: UTF-8, ISO-8859-5
+% Distribution and use is free, also
+% for commercial purposes.
+%
+% TODO:
+% -- fix LC_COLLATE to work correctly for latin alphabet too
+%
+
+LC_IDENTIFICATION
+title      "Serbian locale for Serbia and Montenegro"
+source     "sr_YU locale"
+address    ""
+contact    "Danilo Segan"
+email      "bug-glibc-locales@gnu.org"
+tel        ""
+fax        ""
+language   "Serbian"
+territory  "Serbia and Montenegro"
+revision   "1.2"
+date       "2004-08-09"
+audience   "general"
+application   "GNU locale"
+abbreviation  ""
+%
+category  "i18n:1997";LC_IDENTIFICATION
+category  "i18n:1997";LC_CTYPE
+category  "i18n:1997";LC_COLLATE
+category  "i18n:1997";LC_MONETARY
+category  "i18n:1997";LC_NUMERIC
+category  "i18n:1997";LC_TIME
+category  "i18n:1997";LC_MESSAGES
+category  "i18n:1997";LC_PAPER
+category  "i18n:1997";LC_NAME
+category  "i18n:1997";LC_ADDRESS
+category  "i18n:1997";LC_TELEPHONE
+category  "i18n:1997";LC_MEASUREMENT
+END LC_IDENTIFICATION
+
+LC_CTYPE
+copy "i18n"
+translit_start
+include "translit_combining";""
+
+% cyrillic -> latin
+<U0452> "<U0111>";"<U0064><U006A>" % d-
+<U0453> "<U0067><U006A>" % gj
+<U0455> "<U0064><U007A>" % dz
+<U0458> "<U006A>" % j
+<U0459> "<U006C><U006A>" % lj
+<U045A> "<U006E><U006A>" % nj
+<U045B> "<U0107>";"<U0063>" % c'
+<U045C> "<U006B><U006A>" % kj
+<U045F> "<U0064><U017E>";"<U0064><U007A>" % dz<
+
+<U0402> "<U0110>";"<U0044><U006A>" % D-
+<U0403> "<U0047><U006A>" % Gj
+<U0405> "<U0044><U007A>" % Dz
+<U0408> "<U004A>" % J
+<U0409> "<U004C><U006A>" % Lj
+<U040A> "<U004E><U006A>" % Nj
+<U040B> "<U0106>";"<U0043>" % C'
+<U040C> "<U004B><U006A>" % Kj
+<U040F> "<U0044><U017E>";"<U0064><U007A>" % Dz<
+
+<U0430> "<U0061>"
+<U0431> "<U0062>"
+<U0446> "<U0063>"
+<U0434> "<U0064>"
+<U0435> "<U0065>"
+<U0444> "<U0066>"
+<U0433> "<U0067>"
+<U0445> "<U0068>"
+<U0438> "<U0069>"
+<U043A> "<U006B>"
+<U043B> "<U006C>"
+<U043C> "<U006D>"
+<U043D> "<U006E>"
+<U043E> "<U006F>"
+<U043F> "<U0070>"
+<U0440> "<U0072>"
+<U0441> "<U0073>"
+<U0442> "<U0074>"
+<U0443> "<U0075>"
+<U0432> "<U0076>"
+<U0437> "<U007A>"
+<U0436> "<U017E>";"<U007A>" % z<
+<U0448> "<U0161>";"<U0073>" % s<
+<U0447> "<U010D>";"<U0063>" % c<
+
+<U0410> "<U0041>"
+<U0411> "<U0042>"
+<U0426> "<U0043>"
+<U0414> "<U0044>"
+<U0415> "<U0045>"
+<U0424> "<U0046>"
+<U0413> "<U0047>"
+<U0425> "<U0048>"
+<U0418> "<U0049>"
+<U041A> "<U004B>"
+<U041B> "<U004C>"
+<U041C> "<U004D>"
+<U041D> "<U004E>"
+<U041E> "<U004F>"
+<U041F> "<U0050>"
+<U0420> "<U0052>"
+<U0421> "<U0053>"
+<U0422> "<U0054>"
+<U0423> "<U0055>"
+<U0412> "<U0056>"
+<U0417> "<U005A>"
+<U0416> "<U017D>";"<U005A>" % z<
+<U0428> "<U0160>";"<U0053>" % s<
+<U0427> "<U010C>";"<U0063>" % c<
+
+translit_end
+END LC_CTYPE
+
+
+LC_COLLATE
+
+copy "iso14651_t1"
+
+collating-symbol  <d-z-sh>
+collating-symbol  <l-j-sh>
+collating-symbol  <n-j-sh>
+
+collating-symbol <cacute>
+collating-symbol <ccaron>
+collating-symbol <d//>
+collating-symbol <scaron>
+collating-symbol <zcaron>
+
+collating-symbol <CAP-MIN>
+collating-symbol <MIN-CAP>
+
+reorder-after <MIN>
+<MIN-CAP>
+reorder-after <CAP>
+<CAP-MIN>
+
+reorder-after <c>
+<cacute>
+<ccaron>
+<d>
+<d-z-sh>
+<d//>
+reorder-after <l>
+<l-j-sh>
+reorder-after <n>
+<n-j-sh>
+reorder-after <s>
+<scaron>
+reorder-after <z>
+<zcaron>
+
+reorder-after <U0063>
+<U0107>	<cacute>;<PCT>;<MIN>;IGNORE
+<U010D>	<ccaron>;<PCT>;<MIN>;IGNORE
+reorder-after <U0043>
+<U0106>	<cacute>;<PCT>;<CAP>;IGNORE
+<U010C>	<ccaron>;<PCT>;<CAP>;IGNORE
+
+reorder-after <U0064>
+<d-z<>	<d-z-sh>;<PCT>;<MIN>;IGNORE
+<dz<>	<d-z-sh>;<PCT>;<MIN>;IGNORE
+<d-Z<>	<d-z-sh>;<PCT>;<MIN-CAP>;IGNORE
+<U0111>	<d//>;<PCT>;<MIN>;IGNORE
+reorder-after <U0044>
+<D-Z<>	<d-z-sh>;<PCT>;<CAP>;IGNORE
+<DZ<>	<d-z-sh>;<PCT>;<CAP>;IGNORE
+<D-z<>	<d-z-sh>;<PCT>;<CAP-MIN>;IGNORE
+<Dz<>	<d-z-sh>;<PCT>;<CAP-MIN>;IGNORE
+<U0110>	<d//>;<PCT>;<CAP>;IGNORE
+
+reorder-after <U006C>
+<l-j>	<l-j-sh>;<PCT>;<MIN>;IGNORE
+<lj>	<l-j-sh>;<PCT>;<MIN>;IGNORE
+<l-J>	<l-j-sh>;<PCT>;<MIN-CAP>;IGNORE
+reorder-after <U004C>
+<L-J>	<l-j-sh>;<PCT>;<CAP>;IGNORE
+<LJ>	<l-j-sh>;<PCT>;<CAP>;IGNORE
+<L-j>	<l-j-sh>;<PCT>;<CAP-MIN>;IGNORE
+<Lj>	<l-j-sh>;<PCT>;<CAP-MIN>;IGNORE
+
+reorder-after <U006E>
+<n-j>	<n-j-sh>;<PCT>;<MIN>;IGNORE
+<nj>	<n-j-sh>;<PCT>;<MIN>;IGNORE
+<n-J>	<n-j-sh>;<PCT>;<MIN-CAP>;IGNORE
+reorder-after <U004E>
+<N-J>	<n-j-sh>;<PCT>;<CAP>;IGNORE
+<NJ>	<n-j-sh>;<PCT>;<CAP>;IGNORE
+<N-j>	<n-j-sh>;<PCT>;<CAP-MIN>;IGNORE
+<Nj>	<n-j-sh>;<PCT>;<CAP-MIN>;IGNORE
+
+reorder-after <U0073>
+<U0161>	<scaron>;<PCT>;<MIN>;IGNORE
+reorder-after <U0053>
+<U0160>	<scaron>;<PCT>;<CAP>;IGNORE
+
+reorder-after <U007A>
+<U017E>	<zcaron>;<PCT>;<MIN>;IGNORE
+reorder-after <U005A>
+<U017D>	<zcaron>;<PCT>;<CAP>;IGNORE
+
+reorder-end
+
+END LC_COLLATE
+
+LC_MONETARY
+int_curr_symbol           "<U0043><U0053><U0044><U0020>"
+currency_symbol           "<U0434><U0438><U043D>"
+mon_decimal_point         "<U002C>"
+mon_thousands_sep         "<U002E>"
+mon_grouping              3;3
+positive_sign             ""
+negative_sign             "<U002D>"
+int_frac_digits           2
+frac_digits               2
+p_cs_precedes             0
+p_sep_by_space            1
+n_cs_precedes             0
+n_sep_by_space            1
+p_sign_posn               1
+n_sign_posn               1
+END LC_MONETARY
+
+LC_NUMERIC
+decimal_point             "<U002C>"
+thousands_sep             ""
+grouping                  0;0
+END LC_NUMERIC
+
+LC_TIME
+
+abday   "<U043D><U0435><U0434>";"<U043F><U043E><U043D>";/
+        "<U0443><U0442><U043E>";"<U0441><U0440><U0435>";/
+        "<U0447><U0435><U0442>";"<U043F><U0435><U0442>";/
+        "<U0441><U0443><U0431>"
+day     "<U043D><U0435><U0434><U0435><U0459><U0430>";/
+        "<U043F><U043E><U043D><U0435><U0434><U0435><U0459><U0430><U043A>";/
+        "<U0443><U0442><U043E><U0440><U0430><U043A>";/
+        "<U0441><U0440><U0435><U0434><U0430>";/
+        "<U0447><U0435><U0442><U0432><U0440><U0442><U0430><U043A>";/
+        "<U043F><U0435><U0442><U0430><U043A>";/
+        "<U0441><U0443><U0431><U043E><U0442><U0430>"
+abmon   "<U0458><U0430><U043D>";"<U0444><U0435><U0431>";/
+        "<U043C><U0430><U0440>";"<U0430><U043F><U0440>";/
+        "<U043C><U0430><U0458>";"<U0458><U0443><U043D>";/
+        "<U0458><U0443><U043B>";"<U0430><U0432><U0433>";/
+        "<U0441><U0435><U043F>";"<U043E><U043A><U0442>";/
+        "<U043D><U043E><U0432>";"<U0434><U0435><U0446>"
+mon     "<U0458><U0430><U043D><U0443><U0430><U0440>";/
+        "<U0444><U0435><U0431><U0440><U0443><U0430><U0440>";/
+        "<U043C><U0430><U0440><U0442>";/
+        "<U0430><U043F><U0440><U0438><U043B>";/
+        "<U043C><U0430><U0458>";/
+        "<U0458><U0443><U043D>";/
+        "<U0458><U0443><U043B>";/
+        "<U0430><U0432><U0433><U0443><U0441><U0442>";/
+        "<U0441><U0435><U043F><U0442><U0435><U043C><U0431><U0430><U0440>";/
+        "<U043E><U043A><U0442><U043E><U0431><U0430><U0440>";/
+        "<U043D><U043E><U0432><U0435><U043C><U0431><U0430><U0440>";/
+        "<U0434><U0435><U0446><U0435><U043C><U0431><U0430><U0440>"
+
+d_t_fmt "<U0025><U0041><U002C><U0020><U0025><U0064><U002E><U0020>/
+<U0025><U0042><U0020><U0025><U0059><U002E><U0020><U0025><U0054>/
+<U0020><U0025><U005A>"
+d_fmt   "<U0025><U0064><U002E><U0025><U006D><U002E><U0025><U0059><U002E>"
+t_fmt   "<U0025><U0054>"
+am_pm   "";""
+t_fmt_ampm "<U0025><U0054>"
+date_fmt  "<U0025><U0061><U002C><U0020><U0025><U0065><U002E><U0020>/
+<U0025><U0062><U0020><U0025><U0059><U002E><U0020><U0020><U0025><U0048>/
+<U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A><U000A>"
+week    7;19971201;4
+first_weekday 2
+first_workday 2
+END LC_TIME
+
+LC_MESSAGES
+yesexpr "<U005E><U005B><U0414><U0434><U0044><U0064><U0059><U0079><U005D>"
+noexpr  "<U005E><U005B><U041D><U043D><U004E><U006E><U005D>"
+END LC_MESSAGES
+
+LC_PAPER
+height   297
+width    210
+END LC_PAPER
+
+LC_NAME
+name_fmt    "<U0025><U0064><U0025><U0074><U0025><U0067><U0025><U0074>/
+<U0025><U006D><U0025><U0074><U0025><U0066>"
+name_gen    ""
+name_mr     "<U0433><U002D><U0434><U0438><U043D>"
+name_mrs    "<U0433><U002D><U0452><U0430>"
+name_miss   "<U0433><U002D><U0452><U0438><U0446><U0430>"
+name_ms     "<U0433><U002D><U0452><U0430>"
+END LC_NAME
+
+LC_ADDRESS
+postal_fmt    "<U0025><U0066><U0025><U004E><U0025><U0061><U0025><U004E>/
+<U0025><U0064><U0025><U004E><U0025><U0062><U0025><U004E><U0025><U0073>/
+<U0020><U0025><U0068><U0020><U0025><U0065><U0020><U0025><U0072><U0025>/
+<U004E><U0025><U0025><U007A><U0020><U0025><U0054><U0025>/
+<U004E><U0025><U0063><U0025><U004E>"
+country_ab2   "<U0043><U0053>"
+country_ab3   "<U0053><U0043><U0047>"
+% FIXME: country_post is probably incorrect
+country_post  "<U0053><U0043><U0047>"
+country_name  "<U0421><U0440><U0431><U0438><U0458><U0430><U0020><U0438>/
+<U0020><U0426><U0440><U043D><U0430><U0020><U0413><U043E><U0440><U0430>"
+country_num   891
+country_car   "<U0053><U0043><U0047>"
+% FIXME: ISBN code is what? "86" that preceedes all the numbers?
+country_isbn  "<U0038><U0036>"
+lang_name     "<U0441><U0440><U043F><U0441><U043A><U0438>"
+lang_term     "<U0073><U0072><U0070>"
+lang_lib      "<U0073><U0063><U0063>"
+lang_ab	      "<U0073><U0072>"
+END LC_ADDRESS
+
+LC_TELEPHONE
+tel_int_fmt    "<U002B><U0025><U0063><U0020><U0025><U0061><U0020><U0025>/
+<U006C>"
+tel_dom_fmt    "<U0025><U0041><U0020><U0025><U006C>"
+int_prefix     "<U0033><U0038><U0031>"
+int_select     "<U0039><U0039>"
+END LC_TELEPHONE
+
+LC_MEASUREMENT
+measurement    1
+END LC_MEASUREMENT
